data/pigui/modules/saveloadgame.lua

```diff
@@ -129,6 +127,7 @@ ui.saveLoadWindow = ModalWindow.New("LoadGame", function()
 
 	showSaveFiles()
 end, function (self, drawPopupFn)
+	ui.setNextWindowSize(ui.rescaleUI(Vector2(350, 400), Vector2(1920,1080)), "Always")
```

Comment:
```suggestion
	ui.setNextWindowSize(ui.rescaleUI(Vector2(640, 400), Vector2(1920,1080)), "Always")
```

I'd prefer the save/load dialog be wider than it is tall as a point of personal preference. This value is chosen for being golden-rule proportionate to 400, but is otherwise arbitrary.

Looks like a very straightforward PR otherwise, thanks! The ideal goal is to redesign this dialog so it fits our UI theming a little better, but until vakhoir comes back or I can drag myself out of the engine internals, this is a very acceptable solution. :smile:

Reply:
I'm not opposed to that, but it will change the button layout (they'll be grouped to the left), so I'll have to tweak them a bit too I suspect. I'll look at it.